android/build.gradle

```diff
@@ -56,6 +56,6 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'  // From node_modules
-    implementation 'net.openid:appauth:0.8.1'
+    implementation 'net.openid:appauth:0.9.0'
```

Comment:
needed?

Reply:
Looks like `0.9.0` adds some changes to the end session API 

https://github.com/openid/AppAuth-Android/tags